resources/test/fixtures/B006.py

```diff
@@ -0,0 +1,61 @@
+import collections
```

Comment:
Could we instead use the test suite from flake8-bugbear? https://github.com/PyCQA/flake8-bugbear/blob/main/tests/b006_b008.py